aiven/client/cli.py

```diff
@@ -1235,6 +1238,130 @@ def service__get(self):
+    @arg("-ww", "--overwrite", action="store_true", help="Save and overwrite certificate and key files")
+    def service__connection_info__kafkacat(self):
+        """kafkacat command string"""
+        service = self.client.get_service(project=self.get_project(), service=self.args.service_name)
+        store = self._get_store_from_args()
+
+        if self.args.kafka_authentication_method == "certificate":
+            ci = KafkaCertificateConnectionInfo.from_service(service, route=self.args.route, username=self.args.username)
+            cmd = ci.kafkacat(
+                store,
+                get_project_ca=self._get_project_ca,
+                ca_path=self.args.ca_path,
+                client_cert_path=self.args.client_cert_path,
+                client_key_path=self.args.client_key_path,
+            )
+        elif self.args.kafka_authentication_method == "sasl":
+            ci = KafkaSASLConnectionInfo.from_service(service, route=self.args.route, username=self.args.username)
+            cmd = ci.kafkacat(
+                store,
+                get_project_ca=self._get_project_ca,
+                ca_path=self.args.ca_path,
+            )
+        else:
+            raise NotImplementedError(self.args.kafka_authentication_method)
+
+        print(" ".join(cmd))
+
+    def _get_usage_from_args(self):
+        if self.args.usage is None:
+            usage = "replica" if self.args.replica else "primary"
+        else:
+            if self.args.replica:
+                raise argx.UserError("--usage and --replica cannot be used simultaneously")
+            usage = self.args.usage
+        return usage
+
+    @arg.project
+    @arg.service_name
+    @arg("-r", "--route", choices=("dynamic", "privatelink", "public"), default="dynamic")
+    @arg("--usage", choices=("primary", "replica"))
+    @arg("--replica", action="store_true")
+    @arg("-u", "--username", default="avnadmin")
+    @arg("-d", "--dbname", default="defaultdb")
+    @arg("--sslmode", default="require")
+    def service__connection_info__psql(self):
+        """psql command string"""
+        service = self.client.get_service(project=self.get_project(), service=self.args.service_name)
+
+        ci = PGConnectionInfo.from_service(
+            service,
+            route=self.args.route,
+            usage=self._get_usage_from_args(),
+            username=self.args.username,
+            dbname=self.args.dbname,
+            sslmode=self.args.sslmode,
+        )
+        cmd = ci.psql()
+        print(" ".join(cmd))
+
+    @arg.project
+    @arg.service_name
+    @arg("-r", "--route", choices=("dynamic", "privatelink", "public"), default="dynamic")
+    @arg("--usage", choices=("primary", "replica"))
+    @arg("--replica", action="store_true")
+    @arg("-u", "--username", default="avnadmin")
+    @arg("-d", "--dbname", default="defaultdb")
+    @arg("--sslmode", default="require")
```

Comment:
Having `choices` defined for this would be more user-friendly

Reply:
Added `choices`